package/open-remarkable-shutdown/remarkable-reboot-override.conf

```
@@ -0,0 +1,6 @@
+[Unit]
+After=launcher.service
+
+[Service]
+ExecStart=
+ExecStart=/opt/bin/remarkable-splash /usr/share/remarkable/rebooting.png
```

In the stock remarkable-reboot.service definition, the ExecStart entry is prefixed with a dash so that errors from the splash process [are ignored](https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStart=). Should we also do that here?
```suggestion
ExecStart=-/opt/bin/remarkable-splash /usr/share/remarkable/rebooting.png
```

---

Yes, good catch